lighthouse-core/gather/gatherers/dobetterweb/password-inputs-with-prevented-paste.js

```diff
@@ -23,8 +23,8 @@ function findPasswordInputsWithPreventedPaste() {
       )
     )
     .map(passwordInput => ({
-      // @ts-expect-error - getOuterHTMLSnippet put into scope via stringification
-      snippet: getOuterHTMLSnippet(passwordInput),
+      // @ts-expect-error - getNodeInfo put into scope via stringification
+      ...getNodeInfo(passwordInput),
```

Comment:
seems like we don't need the spread at this point and can just use `getNodeInfo` directly?